sonar-python-plugin/src/main/java/org/sonar/plugins/python/indexer/SonarQubePythonIndexer.java

```diff
@@ -172,10 +183,7 @@ public void computeGlobalSymbols(List<PythonInputFile> files, SensorContext cont
     if (caching.isCacheEnabled()) {
       saveGlobalSymbolsInCache(files);
       saveMainFilesListInCache(new HashSet<>(inputFileToFQN.values()));
-      Set<String> stubModules = TypeShed.stubModules();
-      if (!stubModules.isEmpty()) {
-        caching.writeTypeshedModules(stubModules);
-      }
+      // Information on used Typeshed stubs needs to be done at the end of the analysis, as it is not computed during indexing anymore
```

Comment:
I appreciate this comment :smile: I wouldn't have understand really what s going on without it :+1: